packages/vats/src/core/utils.js

```diff
@@ -221,6 +221,7 @@ export const makePromiseSpaceForNameHub = (nameAdmin, log = noop) => {
         logHooks.onAddKey(name);
       },
       onResolve: (name, valueP) => {
+        void E(nameAdmin).reserve(name);
```

Comment:
interesting solution

Reply:
I will add a test for this fix.

Reply:
Turns out it wasn't necessary... the `onAddKey` does a reserve before `onResolve` is called.